youtube_dl/utils.py

```diff
@@ -4262,16 +4263,38 @@ def try_get(src, getter, expected_type=None):
                 return v
 
 
-def merge_dicts(*dicts):
+def merge_dicts(*dicts, **kwargs):
+    """
+        Merge the `dict`s in `dicts` using the first valid value for each key.
+        Normally valid: not None and not an empty string
+
+        Keyword-only args:
+        unblank:    allow empty string if False (default True)
+        rev:        merge dicts in reverse order (default False)
```

Comment:
I don't get the point. Instead of `unblank=False`, why not just do `**d1, **d2`? And instead of `rev=True`, you can just pass the dict in reverse order, no?

Reply:
1. 2.7 knows not `{**d1, **d2}` (`**` is only expanded in the function call, I think).
2. (Hence) `rev` allows the list of `dict`s to be left in the same order when converting from `{**d1, **d2}` and minimising diffs.

There is some logic to this, given point 1.